### PR TITLE
[RetroPlayer] Adjust game loop

### PR DESCRIPTION
**Pull request by GTechAlpha**
## Description
Accounts for sporadic variations and prevents indefinite catch-up.

## Motivation and Context
Discovered inconsistent and undesired timing during testing.

## How Has This Been Tested?
PCSX, GenesisPlus, Nestopia

## Types of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: xbmc/games/addons/playback/GameLoop.cpp [hunk 1]
```diff
@@ -58,79 +59,62 @@ void CGameLoop::Stop()
...
+        m_lastFrameMs += FrameTimeMs();
+        m_adjustTime = m_lastFrameMs - NowMs();
+      }
+      else
+      {
+        m_lastFrameMs = NowMs();
+        m_adjustTime = 0.0;
+      }
+
+      CLog::Log(LOGDEBUG, "[RetroPlayer][GameLoop] adjust time = %0.4f", m_adjustTime);
```
Comment (garbear) [inline xbmc/games/addons/playback/GameLoop.cpp]: We should avoid logging for every frame before this hits master

**Comment by garbear** [comment]
The new timing looks good. I tested and gameplay was smooth. I didn't notice the glitches on my i7, but I haven't tested on embedded in a while.

File: xbmc/games/addons/playback/GameLoop.cpp (hunk 1 above)
Comment (GTechAlpha) [reply xbmc/games/addons/playback/GameLoop.cpp]: Agreed. Intended to remove this.

**Comment by GTechAlpha** [comment]
Updated. Should be good to go.

**Pull request merged**